### PR TITLE
adding to the EC2 section

### PR DESCRIPTION
**Pull request by em0ney**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -707,6 +707,7 @@ EC2
...
 -	The term “EC2” is sometimes used to refer to the servers themselves, but technically refers more broadly to a whole collection of supporting services, too, like load balancing (CLBs/ALBs), IP addresses (EIPs), bootable images (AMIs), security groups, and network drives (EBS) (which we discuss individually in this guide).
+-	💸[EC2 pricing](https://aws.amazon.com/ec2/pricing/) is a complicated topic. It can range from free (on the [AWS free tier](https://aws.amazon.com/free/)) to a lot, depending on your usage of course. Pricing is by instance type, by hour and changes depending on AWS region and whether you are purchasing your instances [on-demand](https://aws.amazon.com/ec2/pricing/on-demand/), on the [spot-market](https://aws.amazon.com/ec2/spot/) or pre-purchasing ([reserved instances](https://aws.amazon.com/ec2/pricing/reserved-instances/)). The website [http://www.ec2instances.info/](http://www.ec2instances.info/) is particularly good for comparing instance types.
```
Comment (jlevy) [inline README.md]: Boldface and forward link to our cost management section:
[**EC2 pricing**](https://aws.amazon.com/ec2/pricing/) and [**cost management**](#ec2-cost-management)  are complicated topics.

Let's drop the last sentence about ec2instances.info as it's already linked above and immediately below.

Nits:
Remove "of course".
"on-demand" -> "On-Demand" (AWS usage)
"spot-market" -> "Spot market" (AWS usage)
"reserved instances" -> "Reserved Instances" (AWS usage)

File: README.md [hunk 2]
```diff
@@ -722,6 +723,7 @@ EC2
...
 	-	🔹Use the list at [**ec2instances.info**](http://www.ec2instances.info/) to review costs and features. [Amazon’s own list](https://aws.amazon.com/ec2/instance-types/) of instance types is hard to use, and doesn’t list features and price together, which makes it doubly difficult.
 	-	Prices vary a lot, so use [**ec2instances.info**](http://www.ec2instances.info/) to determine the set of machines that meet your needs and [**ec2price.com**](http://ec2price.com/) to find the cheapest type in the region you’re working in. Depending on the timing and region, it might be much cheaper to rent an instance with *more* memory or CPU than the bare minimum.
+  - Turn off your instances when they aren't in use. For multiple workloads/environments, you don't need your EC2 instances on 24/7. Given that costs are calculated based on hourly usage, this is a simple mechanism for cost savings. This can be achieved using [Lambda and CloudWatch](https://aws.amazon.com/premiumsupport/knowledge-center/start-stop-lambda-cloudwatch/), an open source solution like [Scalr](https://github.com/Scalr/scalr) or a SaaS provider like [GorillaStack](www.gorillastack.com).
```
Comment (jlevy) [inline README.md]: Move this to the "EC2 Cost Management" section we already have.

Slight rephrase:

"For multiple workloads/environments, you don't need your EC2 instances on 24/7. Given that costs are calculated based on hourly usage, this is a simple mechanism for cost savings."
-> "For many situations such as testing or staging resources, you may not need your instances on 24/7, and you won’t need to pay the EC2 hourly cost when they are suspended."

Boldface **Turn off**

GorillaStack link is missing https://

Probably we should also mention parkmycloud.com ? Has anyone used it?

Comment (max-zanko) [inline README.md]: In addition to what @jlevy already commented on, this tip should start with something like "In general, use spot instances whenever possible. If using On-Demand instances is unavoidable, consider launching them only when needed..."

File: README.md [hunk 3]
```diff
@@ -722,6 +723,7 @@ EC2
...
 	-	Prices vary a lot, so use [**ec2instances.info**](http://www.ec2instances.info/) to determine the set of machines that meet your needs and [**ec2price.com**](http://ec2price.com/) to find the cheapest type in the region you’re working in. Depending on the timing and region, it might be much cheaper to rent an instance with *more* memory or CPU than the bare minimum.
+  - **Turn off** your instances when they aren't in use. For many situations such as testing or staging resources, you may not need your instances on 24/7, and you won't need to pay EC2 hourly costs when they are suspended. Given that costs are calculated based on hourly usage, this is a simple mechanism for cost savings. This can be achieved using [Lambda and CloudWatch](https://aws.amazon.com/premiumsupport/knowledge-center/start-stop-lambda-cloudwatch/), an open source solution like [Scalr](https://github.com/Scalr/scalr) or a SaaS provider like [GorillaStack](https://www.gorillastack.com).
```
Comment (max-zanko) [inline README.md]: Probably worth mentioning that you can only turn off the EBS-rooted instances. An instance with ephemeral root volume can only be terminated and will lose all the data.
Also, fix the apostrophes (' > ’).

**Comment by jlevy** [comment]
LGTM! Thanks @em0ney !

**Pull request merged**